src/lib/form-field/form-field-input.scss

```diff
@@ -127,13 +127,6 @@ textarea.mat-input-element {
   }
 }
 
-textarea.mat-input-element {
-  // The 2px padding prevents scrollbars from appearing on Chrome even when they aren't needed.
```

Comment:
It seems like these changes were added to fix a different issue. @mmalerba do you know whether they can removed now? You show up on the blame for the changes.

Reply:
Removing the styles ends up with a wrong height on my end as well @mmalerba. It also shows scrollbars when it isn't supposed to.

Reply:
@mmalerba I am using Windows 10 Pro at standard zoom level.